### ISSUE TITLE
Atom linter does not install dependencies

### ISSUE THREAD

**Issue by MicFin** [ga-wdi-boston/orientation#252]
Depends on dependencies package, but pop ups requested to be accepted need to be accepted

**Comment by gaand** [comment]
@MicFin Is this about linter version 2 (it's new) ❓

**Comment by jrhorn424** [comment]
Yes. Specifically, after installation, there will be prompts in Atom that ask if we wish to install dependencies. Developers should choose "Yes". I believe one of the dependencies, for example, is "intentions".
